src/clients/user-client/src/routes/(auth)/me.tsx

```
@@ -76,14 +89,75 @@ export function SessionCard({
+              <ItemTitle>Stripe Onboarding</ItemTitle>
+              <ItemDescription>
+                Complete your Stripe onboarding to withdraw funds
+              </ItemDescription>
+            </ItemContent>
+          </Item>
+        </Link>
+
+        <Item
+          className="w-full items-stretch rounded-none"
+          onClick={() => logOutMutation.mutate()}
+          role="button"
```

Biome (2.1.2)

[error] The elements with this role can be changed to the following elements:
<button>

For examples and more information, see WAI-ARIA Roles

(lint/a11y/useSemanticElements)